.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
WTF?! Please add to git ignore or delete from source dir

Reply:
thx. Forget add .vscode dir to gitignore. Fix in latest commit